ortools/math_opt/solvers/highs_solver_test.cc

```diff
@@ -164,8 +164,8 @@ INSTANTIATE_TEST_SUITE_P(HighsLpModelSolveParametersTest,
                              /*supports_duals=*/true,
                              /*supports_primal_only_warm_starts=*/false)));
 
-// MIP hint appears to be supported by Highs::setSolution, this is not yet
-// implemented.
+// Highs::setSolution is implemented, but it only accepts complete solutions.
```

Comment:
We cannot submit code without tests. As commented above, perhaps you can get the partial hint working.

If not, then you need to fork this test:

https://github.com/google/or-tools/blob/stable/ortools/math_opt/solver_tests/ip_model_solve_parameters_tests.cc#L101

in the same file to add a version of the test that has a complete hint.

A fallback would be to add the fork of this test directly to highs_solver_test.cc that does this, if you cannot get it working for every solver.

Hopefully, partial hints are actually supported.